### ISSUE TITLE
Error while fetch v4.1.1 - No matching version found for chalk@4.1.1.

### ISSUE THREAD

**Issue by GauriBhutkar** [chalk/chalk#436]
Seems like there has been a minor version upgrade done some time ago, could you please validate if latest version is published properly?
I have a dependency which uses `^4.1.0`, it is automatically going for `4.1.1`(which is also present on npm).
It's weird why it keeps throwing this error. Started coming only after latest minor version upgrade. `No matching version found for chalk@4.1.1.`

**Comment by Richienb** [comment]
After an npm package is published, it takes a while for npm installations to realise that it exists because of the caching it does. At the time you opened the issue, `chalk@4.1.1` was published 11 hours ago. Try again now or in a day or so. If it still doesn't work, have a look at clearing your npm cache.

**Comment by Qix-** [comment]
Duplicate of #435. Please search next time - it's literally the issue filed directly before this one.
